src/js/page.js

```diff
@@ -81,14 +81,22 @@ export default {
      * @param {jQuery} $e - Element on page to flip to
      */
     flipToPageContaining( $e ) {
+        const e = $e[ 0 ];
         let $closest;
```

Comment:
No need to define here. It's also no longer variable so can be declared with `const = e.cl..` directtly.

Reply:
code updated